db/migrate/20230814094947_create_users.rb

```diff
@@ -8,5 +8,6 @@ def change
 
       t.timestamps
     end
-  end
+  
+    add_index :users, :name, unique: true
 end
```

Comment:
- [ ] There is a missing `end` for the class that is causing the migration to not run successfully. Kindly add the closing `end` for the class so your migration for create_users works fine. 👍🏽

Reply:
Fixed.